cmd/loom/chainconfig/cmd.go

```diff
@@ -214,11 +216,42 @@ func ListFeaturesCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
-			out, err := formatJSON(&resp)
-			if err != nil {
-				return err
+
+			type MaxLength struct {
+				Name        int
+				Status      int
+				Validators  int
+				Height      int
+				Percentage  int
+				BuildNumber int
+			}
+
+			ml := MaxLength{Name: 4, Status: 6, Validators: 10, Height: 6, Percentage: 6, BuildNumber: 5}
+			for _, value := range resp.Features {
+				if len(value.Name) >= ml.Name {
+					ml.Name = len(value.Name)
+				}
+				if len(value.Status.String()) >= ml.Status {
+					ml.Status = len(value.Status.String())
+				}
+				if uintLength(value.BlockHeight) >= ml.Height {
+					ml.Height = uintLength(value.BlockHeight)
+				}
+				if uintLength(value.Percentage) >= ml.Percentage {
+					ml.Percentage = uintLength(value.Percentage)
+				}
+				if uintLength(value.BuildNumber) >= ml.BuildNumber {
```

Comment:
BuildNumber is not gonna exceed 10000 soon, so it is safe to set 5. No need to check it.